src/components/RelationshipPicker.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+export default function RelationshipPicker(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="md">
+      <div className={classes.paper}>
+        <Typography component="h2" variant="h6">
+          Search with a Relationship
+        </Typography>
+        <Typography>Spouse | Father | Mother | Other Person </Typography>
```

Comment:
Eventually these will have to be individual Buttons, won't they?

Reply:
Yes, as per Heather it should add another text field when clicked like here https://www.familysearch.org/search/